docs/lab.md

```diff
@@ -25,6 +25,8 @@ By convention, an autograder accepts an optional `-A` command line argument that
 
 One of the nice properties of Autolab autograders is that they can be written and tested offline, without requiring any interaction with Autolab. Writing autograders is not easy, but the fact that they can be developed offline allows you to develop and test them in your own familiar computing environment.
 
+To format your autoresult feedback provided to the students, use the [formatted feedback feature](/features/formatted-feedback).
```

Comment:
Moved this reminder to use formatted feedback from Troubleshooting to the feature introduction itself.